package.json

```diff
@@ -1,11 +1,12 @@
 {
+    "enableProposedApi": true,
```

Comment:
remember to remove this when VS Code release latest stable version

Reply:
ok